generator/js_generator.cc

```diff
@@ -3446,72 +3442,72 @@ void Generator::GenerateExtension(const GeneratorOptions& options,
 bool GeneratorOptions::ParseFromOptions(
     const std::vector<std::pair<std::string, std::string> >& options,
     std::string* error) {
-  for (int i = 0; i < options.size(); i++) {
-    if (options[i].first == "add_require_for_enums") {
-      if (options[i].second != "") {
+  for (auto option : options) {
```

Comment:
optional: bonus points for destructuring instead

Reply:
Is it okay to require C++17?

Reply:
Good question. Disregard the destructing suggestion.  Let's just change this to const auto& option.